baiji/util/test_reachability.py

```diff
@@ -0,0 +1,9 @@
+import unittest
+
+class TestReachability(unittest.TestCase):
+    def test_internet_reachable(self):
+        # We assume these tests run with a reachable Internet. This test is
```

Comment:
Not all of the tests should require access to the internet though. Most should be able to run by mocking key boto functionality, no? Integration tests would require access to the internet, but I think a test case mixin would be a more appropriate place for this check.

I'm not suggesting you remove the test, just that the spirit of the comment might be changed.